apps/web/components/shared/project-card.tsx

```
@@ -1,96 +1,212 @@
+									<span
+										key={tag.id}
+										className="px-2 py-1 text-xs rounded uppercase"
+										style={{
+											backgroundColor: tag.color?.backgroundColor ?? "",
+											color: tag.color?.textColor ?? "",
+										}}
+									>
+										{tag.text}
+									</span>
+								);
+							}
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)